lib/vagrant/util/downloader.rb

```diff
@@ -203,9 +203,13 @@ def execute_curl(options, subprocess_options, &data_proc)
           @logger.warn("Downloader exit code: #{result.exit_code}")
           parts    = result.stderr.split(/\n*curl:\s+\(\d+\)\s*/, 2)
           parts[1] ||= ""
-          raise Errors::DownloaderError,
-            code: result.exit_code,
-            message: parts[1].chomp
+          if parts[1].include? "416"
+            # All good actually. 416 means there is no mory bytes to download
```

Comment:
@glennpratt Else need to switch from curl to wget which handles this status code appropriately (it exits with 0 code when 416 is returned, unlike curl which delegates handling of that to user level)